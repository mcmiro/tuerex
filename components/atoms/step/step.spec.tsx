import * as React from "react";
import {screen, render} from "@testing-library/react";
import Step from "./";

describe("Test step component", () => {
	it("Should pass when component is defined", () => {
		render(<Step data-testid="step-test" />);

		const step = screen.queryAllByTestId("step-test");
		expect(step).toBeDefined();
		expect(step).toMatchSnapshot();
	});
});
