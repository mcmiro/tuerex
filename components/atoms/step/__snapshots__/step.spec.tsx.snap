// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test step component Should pass when component is defined 1`] = `[]`;
